app/javascript/utils/articles.js

```diff
@@ -0,0 +1,52 @@
+const articles = [
```

Comment:
we don't need this file anymore, as we are fetching the real data from seed.rb.

Reply:
Great observation, thanks!